pytest.ini

```diff
@@ -0,0 +1,2 @@
+[pytest]
+addopts = --tb=short -rxs
```

Comment:
If I remember correctly, I think we can actually add this to `tox.ini` and `py.test`  will read it the same way.